CHANGELOG.md

```diff
@@ -51,6 +51,17 @@ from `algosdk` and sends it to the network.
 - Fix `TxParams.noteb64` encoding - should use base64 decoder rather than TextEncoder.
 - Fix `ed25519verify` opcode implementation. Previously the signature was only checked against the data not the concatenation of "ProgData"||program||data. Additionally test scenarios was added to check the correct implementation. 
 
+### Breaking Changes
+
+#### @algo-builder/runtime
+
+- `executeTx` now returns `TxnReceipt[]` instead of `TxReceipt[]`.
+- `fundLsig` now returns `TxnReceipt[]` instead of `TxReceipt[]`.
+
+#### @algo-builder/web
+
+- `executeTx` promise now returns `TxnReceipt` instead of `algosdk.modelsv2.PendingTransactionResponse`.
```

Comment:
```suggestion
- `executeTx` promise now returns `[TxnReceipt](<link-here>)` instead of `algosdk.modelsv2.PendingTransactionResponse`.
```

Reply:
but if I have to add the link, it will be of this particular branch which we will be deleted after merge, so how to add it?

Reply:
you can just change the branch name in the hyperlink, or directly link to the api docs (main page)